sdk/storage/blobs/src/az_storage_blobs_blob_client.c

```diff
@@ -53,19 +53,19 @@ AZ_NODISCARD az_storage_blobs_blob_client_options az_storage_blobs_blob_client_o
 }
 
 AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
-    az_storage_blobs_blob_client* self,
+    az_storage_blobs_blob_client* client,
```

Comment:
Let's check across the team, if we still use `self` (which is basically "`this`" but does not use a reserved keyword if compiled by a C++ compiler), or named parameters such as `client`. I am fine either way as long as we are consistent. If we decide to not use `self`, let's replace all other instances. My opinion is that `client` is more readable and easier to for the customers, and not using `self` is easier for us when the function does not conceptually aligns to `self`'s "method" in the OOP sense.

Reply:
I agree, I really have come not to like 'self'. When I am reading a function's code, I have no idea what 'self' refers to and since we use it a lot, it has different meaning in different places and it's really hard to manage understand the code base. I'd like to abolish 'self'

Reply:
I prefer client.

Reply:
Ahson, can you please check the code for the instances of using `self` and update them to better names depending on the context?

Reply:
Nit, I'd prefer that cleanup be a separate PR.

Reply:
> Nit, I'd prefer that cleanup be a separate PR.

I agree.